src/frontend/screens/Settings/components/AutoDXVK.tsx

```diff
@@ -16,17 +16,28 @@ const AutoDXVK = () => {
   const home = configStore.get('userHome', '')
   const [winePrefix] = useSetting('winePrefix', `${home}/.wine`)
   const [wineVersion] = useSetting('wineVersion', defaultWineVersion)
+  const [installingDxvk, setInstallingDxvk] = React.useState(false)
 
-  const isProton = wineVersion.type === 'proton'
+  const dontRender =
+    wineVersion.type === 'proton' || wineVersion.type === 'crossover'
 
-  if (isProton) {
+  if (dontRender) {
```

Comment:
Why not just do
```suggestion
  if (wineVersion.type !== 'wine') {
```

Reply:
Ah, the thing here is that I wanted to support a new type called `wineskin`, but the maintainer said that he will change how the wrappers works and will bring all the features to the main bin, I don't know how. so that wont be necessary. So yes, your solution will work in this case.